backend/app/controllers/OdinsonController.scala

```diff
@@ -733,6 +737,46 @@ class OdinsonController @Inject() (config: Config = ConfigFactory.load(), cc: Co
     }
   }
 
+  /** Return all terms for a given field in orthographic order.   *
+    * @param field A token field such as word, lemma, or tag.
+    * @return The complete [[List]] of terms in this field for the current index.
+    */
+  private def fieldVocabulary(field: String): List[String] = {
+    // get terms from the requested field (error if it doesn't exist)
+    val extractorEngine: ExtractorEngine = newEngine()
+    val fields = MultiFields.getFields(extractorEngine.indexReader)
+    val terms = fields.terms(field)
+    val termsEnum = terms.iterator
```

Comment:
per @kwalcock : there exists a converter from a java iterator to a scala iterator (look in collections.javaconverters._ or something)
then just to list